From 7a88d28ea287bd22ef95d8f3b52c193b63f3e51c Mon Sep 17 00:00:00 2001
From: Lauri Kopo <lauri.kopo@jolla.com>
Date: Tue, 28 Jan 2020 10:25:36 +0200
Subject: [PATCH] disable_x_backend

---
 libselinux/src/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libselinux/src/Makefile b/libselinux/src/Makefile
index 7f5a5d74..a404ffd0 100644
--- a/libselinux/src/Makefile
+++ b/libselinux/src/Makefile
@@ -127,6 +127,10 @@ endif
 
 SWIGRUBY = swig -Wall -ruby -o $(SWIGRUBYCOUT) -outdir ./ $(DISABLE_FLAGS)
 
+# Don't build X backend
+DISABLE_FLAGS+= -DNO_X_BACKEND
+SRCS:= $(filter-out label_x.c, $(SRCS))
+
 all: $(LIBA) $(LIBSO) $(LIBPC)
 
 pywrap: all selinuxswig_python_exception.i
-- 
2.24.1

