From eac53a8b28e1027fa5b0f6f2e9f6407bbbd94f5e Mon Sep 17 00:00:00 2001
From: Lauri Kopo <lauri.kopo@jolla.com>
Date: Tue, 28 Jan 2020 10:21:31 +0200
Subject: [PATCH] enable_android_backend

---
 libselinux/src/Makefile | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/libselinux/src/Makefile b/libselinux/src/Makefile
index 7f5a5d74..d9978643 100644
--- a/libselinux/src/Makefile
+++ b/libselinux/src/Makefile
@@ -114,17 +114,6 @@ ifneq (,$(filter i386,$(ARCH)))
 TLSFLAGS += -mno-tls-direct-seg-refs
 endif
 
-ifeq ($(ANDROID_HOST),y)
-DISABLE_FLAGS+= -DNO_MEDIA_BACKEND -DNO_DB_BACKEND -DNO_X_BACKEND \
-	-DBUILD_HOST
-SRCS= callbacks.c freecon.c label.c label_file.c \
-	label_backends_android.c regex.c label_support.c \
-	matchpathcon.c setrans_client.c sha1.c booleans.c
-else
-DISABLE_FLAGS+= -DNO_ANDROID_BACKEND
-SRCS:= $(filter-out label_backends_android.c, $(SRCS))
-endif
-
 SWIGRUBY = swig -Wall -ruby -o $(SWIGRUBYCOUT) -outdir ./ $(DISABLE_FLAGS)
 
 all: $(LIBA) $(LIBSO) $(LIBPC)
-- 
2.24.1

